local_settings.sh.in

```diff
@@ -323,12 +323,16 @@ verify_policies() {
 				break
 			fi
 		done
-		test ${local_failed} -ne 0 && echo "Missing ${module}!"
+		test ${local_failed} -ne 0 && (echo "Missing ${module}!" >/dev/stderr)
```

Comment:
We should use >&2 instead of device files for stderr

Reply:
(also in below block)

Reply:
Right - should have thought about it.